biome.jsonc

```diff
@@ -7,16 +7,22 @@
     "enabled": true,
     "rules": {
       "recommended": true
-    }
+    },
+    "ignore": ["node_modules", "dist", ".next"]
   },
   "formatter": {
     "enabled": true,
     "indentWidth": 2,
-    "indentStyle": "space"
+    "indentStyle": "space",
+    "ignore": ["node_modules", "dist", ".next"]
   },
   "json": {
+    "parser": {
+      "allowTrailingCommas": true
+    },
     "formatter": {
-      "enabled": true
+      "enabled": true,
+      "trailingCommas": "none"
```

Comment:
default is "none", so I don't think it's necessary to add it.

ref: https://biomejs.dev/reference/configuration/#jsonformattertrailingcommas

Reply:
Thanks!!! I also felt this was unnecessary and fixed it!
[fix: delete useless config](https://github.com/coauthors/coauthors/pull/5/commits/e85ed745d6e47dc3d71874f87674db6a96cca82f)